control-plane/stor-port/src/types/v0/store/node.rs

```diff
@@ -152,8 +152,9 @@ pub struct NodeSpec {
     #[serde(default)] // Ensure backwards compatibility in etcd when upgrading.
     cordon_drain_state: Option<CordonDrainState>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    #[serde(default)] // Ensure backwards compatibility in etcd when upgrading.
     node_nqn: Option<HostNqn>,
+    #[serde(default, skip)] // Ensure backwards compatibility in etcd when upgrading.
```

Comment:
why did you remove serde default from above?
Also seems we're not writing to etcd, so I think we can change previous functions where I commented about etcd.

Reply:
Looks like that was a typo, fixing....